runner/materialize.go

```diff
@@ -197,7 +197,13 @@ func (m MaterializeRunner) Run() (types.CompletionWatcher, error) {
 		m.Logger.Infow("Making Local Runner", "name", m.ID.Name, "variant", m.ID.Variant)
 		completionList := make([]types.CompletionWatcher, int(numChunks))
 		for i := 0; i < int(numChunks); i++ {
-			localRunner, err := Create(COPY_TO_ONLINE, serializedConfig)
+			m.Logger.Infow("Creating materialization chunk", "name", m.ID.Name, "variant", m.ID.Variant, "chunkIndex", i, "chunkSize", chunkSize)
+			config.ChunkIdx = int64(i)
+			serializedChunkConfig, err := config.Serialize()
```

Comment:
Was the other config not passing the chunk size in resulting in 0 everytime?

Reply:
Exactly, so while the number of chunks was being correctly evaluated,  the `ChunkIndex` for each one was always 0, which means they all query the first page of data.